src/components/discover-sheet/ListsSection.js

```diff
@@ -0,0 +1,333 @@
+  const initialized = useRef(false);
+  const { allAssets } = useAccountAssets();
+  const { genericAssets } = useSelector(({ data: { genericAssets } }) => ({
+    genericAssets,
+  }));
+  const { colors } = useTheme();
+  const listData = useMemo(() => DefaultTokenLists[network], [network]);
+
+  const assetsSocket = useSelector(
+    ({ explorer: { assetsSocket } }) => assetsSocket
+  );
+  useEffect(() => {
+    if (assetsSocket !== null) {
+      Object.values(listData).forEach(({ tokens }) => {
+        dispatch(emitAssetRequest(tokens));
+        dispatch(emitChartsRequest(tokens));
+      });
+    }
+  }, [assetsSocket, dispatch, listData]);
+
+  const trendingListHandler = useRef(null);
+
+  const updateTrendingList = useCallback(async () => {
+    const tokens = await fetchTrendingAddresses();
+    clearList('trending');
+
+    dispatch(emitAssetRequest(tokens));
+    dispatch(emitChartsRequest(tokens));
+    updateList(tokens, 'trending', true);
+
+    trendingListHandler.current = setTimeout(
+      () => updateTrendingList(),
+      TRENDING_LIST_UPDATE_INTERVAL
+    );
+  }, [clearList, dispatch, updateList]);
+
+  const handleSwitchList = useCallback(
+    (id, index) => {
+      LayoutAnimation.configureNext(
+        LayoutAnimation.create(200, 'easeInEaseOut', 'opacity')
+      );
+      setSelectedList(id);
+      listRef.current?.scrollToIndex({
+        animated: true,
+        index,
+        viewPosition: 0.5,
+      });
+    },
+    [setSelectedList]
+  );
+
+  useEffect(() => {
+    if (ready && !initialized.current) {
+      ready && updateTrendingList();
+      const currentListIndex = findIndex(
+        lists,
+        list => list.id === selectedList
+      );
+      setTimeout(() => {
+        handleSwitchList(lists[currentListIndex].id, currentListIndex);
+      }, 300);
+      initialized.current = true;
+    }
+    return () => {
+      clearTimeout(trendingListHandler.current);
+    };
+  }, [
+    ready,
+    clearList,
+    dispatch,
+    updateList,
+    updateTrendingList,
+    lists,
+    selectedList,
+    handleSwitchList,
+  ]);
+
+  const listItems = useMemo(() => {
+    let items = [];
+    if (selectedList === 'favorites') {
+      items = favorites.map(
+        item =>
+          ethereumUtils.getAsset(allAssets, toLower(item.address)) ||
+          ethereumUtils.formatGenericAsset(
+            genericAssets[toLower(item.address)],
+            nativeCurrency
+          )
+      );
+    } else {
+      if (!lists?.length) return [];
+      const currentList = lists.find(list => list.id === selectedList);
+      if (!currentList) {
+        return [];
+      }
+
+      items = currentList.tokens.map(
+        address =>
+          ethereumUtils.getAsset(allAssets, toLower(address)) ||
+          ethereumUtils.formatGenericAsset(
+            genericAssets[toLower(address)],
+            nativeCurrency
+          )
+      );
+    }
+
+    return items.filter(item => {
```

Comment:
forgot to add: can we do something like: `return items.filter(item => !!item.symbol && Number(item.price?.value) > 0);`

Reply:
I think I left this logic cause I was logging. Yeah, changing it like you said.